content-repo/extra-docs/articles/Email-Communication-Readme.md

```diff
@@ -0,0 +1,195 @@
+---
+id: email-communication
+title: Email Communication
+description: Communication across and between departments is a vital component of collecting information, and managing and remediating security events. The Email Communication content pack enables security teams to automate and streamline the communication and notification process with users across your organization via email.
+ 
+---
+This Email Communication pack enables security teams to reply to an email as part of an incident. The SOC team is able to communicate directly through the Cortex XSOAR platform as part of the remediation process. 
+
+
+
+
+## Pack Workflow
+When an email is sent to the email address configured in your email integration, your email listener fetches the incoming email in Cortex XSOAR. 
+
+If the email is in response to an existing incident, an 8-digit number will appear next to the subject of the email, for example <93075875> Incident1.
+
+The pre-process script searches for the 8-digit number in the subject of the email to link to the incident. 
```

Comment:
The script does not search for 8-digit number, but with an `#` followed by an incident id, foe examples `#1234`

Reply:
@omercnet We have an enhancement for that, will be release soon. https://github.com/demisto/content/pull/10222